File: JavaCC/ArithmeticExpressions/Sum.java
public class Sum extends Expr {
	Expr mLhs;
	Expr mRhs;
	
	public Sum(Expr lhs, Expr rhs) {
		mLhs = lhs;
		mRhs = rhs;
	}
	public Sum diff(Variable x)	{
		return new Sum(mLhs.diff(x), mRhs.diff(x));
	}
	public String toString() {
		return mLhs.toString() + " + " + mRhs.toString();
	}
}

File: Python/Chapter-11/sum.c
#import "stdio.h"

int sum(int n) {
    int s;
    s = 0;
    while (n != 0) {
        s = s + n;
        n = n - 1;    
    };
    return s;
}

int main() {
    printf("%d\n", sum(6*6));
    return 0;
}
